service/scaladsl/api/src/main/scala/com/lightbend/lagom/scaladsl/api/ServiceLocator.scala

```diff
@@ -39,6 +47,16 @@ trait ServiceLocator {
   def locate(name: String, serviceCall: Descriptor.Call[_, _]): Future[Option[URI]]
```

Comment:
The clients use `doWithService`, but I also see that the default implementation in `CircuitBreakingServiceLocator` still calls through to `locate`. I think we'll want to change that, but maybe not in this pull request.